jsre/ethereum_js.go

```diff
@@ -5562,10 +5562,18 @@ var methods = function () {
         inputFormatter: [formatters.inputAddressFormatter]
     });
 
+    var importAccount = new Method({
+        name: 'importAccount',
+        call: 'personal_importAccount',
+        params: 2,
+        inputFormatter: [null, null]
+    });
```

Comment:
web3.js extensions should be placed in [package web3ext](https://github.com/ethereum/go-ethereum/blob/develop/internal/web3ext/web3ext.go).